Modules/RPC.cs

```diff
@@ -22,6 +22,7 @@ enum CustomRPC : byte // 193/255 USED
     VersionCheck = 80,
     RequestRetryVersionCheck = 81,
     SyncCustomSettings = 100, // AUM use 101 rpc
+    BetterCheck = 150, // BetterAmongUs (BAU) RPC, This is sent to allow other BAU users know who's using BAU!
```

Comment:
I mean set `BetterCheck` after `SetDeathReason` and set 103 number, lol

Reply:
I moved it up to the top so it doesn't affect any other RPC, but other than that there's not much I can do, it would cause issues if I change the RPC number from 150